src/app/zap-templates/zcl/data-model/chip/application-basic-cluster.xml

```diff
@@ -24,32 +31,23 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>This cluster provides information about an application running on a TV or media player device which is represented as an endpoint.</description>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
     <attribute side="server" code="0x0000" name="VendorName" define="APPLICATION_VENDOR_NAME" type="char_string" length="32" optional="true">
       <optionalConform/>
     </attribute>
     <attribute side="server" code="0x0001" name="VendorID" define="APPLICATION_VENDOR_ID" type="vendor_id" optional="true">
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0002" name="ApplicationName" define="APPLICATION_NAME" type="long_char_string" length="256">
-      <mandatoryConform/>
-    </attribute>
+    <attribute side="server" code="0x0002" name="ApplicationName" define="APPLICATION_NAME" type="long_char_string" length="256"/>
     <attribute side="server" code="0x0003" name="ProductID" define="APPLICATION_PRODUCT_ID" type="int16u" optional="true">
       <optionalConform/>
     </attribute>
-    <attribute side="server" code="0x0004" name="Application" define="APPLICATION_APP" type="ApplicationStruct">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0005" name="Status" define="APPLICATION_STATUS" type="ApplicationStatusEnum" default="0x01" min="0x00" max="0xFF">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0006" name="ApplicationVersion" define="APPLICATION_VERSION" type="char_string" length="32">
-      <mandatoryConform/>
-    </attribute>
+    <attribute side="server" code="0x0004" name="Application" define="APPLICATION_APP" type="ApplicationStruct"/>
+    <attribute side="server" code="0x0005" name="Status" define="APPLICATION_STATUS" type="ApplicationStatusEnum" max="0x03" default="0x01"/>
+    <attribute side="server" code="0x0006" name="ApplicationVersion" define="APPLICATION_VERSION" type="char_string" length="32"/>
     <attribute side="server" code="0x0007" name="AllowedVendorList" define="APPLICATION_ALLOWED_VENDOR_LIST" type="array" entryType="vendor_id" length="32">
       <access op="read" role="administer"/>
-      <mandatoryConform/>
     </attribute>
-
   </cluster>
 
   <!-- ZAP does not support two identically-named structs in different clusters yet, so we just reuse the struct definition in Application Launcher
```

Comment:
This commented-out bit seems to not be true anymore, right?

Reply:
I'm honestly not sure anymore, I was expecting that ZAP was able to handle this scenario but recently tried to regen the "Mode" clusters (e.g. LaundryWasher_Mode) and still got a ZAP error during generation because of repeated structs. I'm retaking the Alchemy bits so I will probably have to take another look at this.

Reply:
It seems to still work, so not sure about ZAP and structs. Probably this fails only for the enums and having repeated structs is OK now. Will have to confirm this with a cluster with shared enums.